apps/site-nft-landing/src/fr.html

```diff
@@ -38,7 +38,16 @@
 <body>
   <div class="content-container">
 
-    <div class="logo"></div>
+    <div class='header'>
+      <div class="logo">
+        <img src='./images/Logo_green.svg' />
+      </div>
+        <a  class='translate' href="./en.html">
+          <span>EN</span>
```

Comment:
The prefinery-form-embed is not translated

Reply:
I know, I'm not sure we can translate it.  I'll create an issue for it, but it might have to leave it for now